src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -5498,6 +5619,108 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/sites/{siteId}/groups/{groupId}':
+    get:
+      tags:
+        - sites
+      summary: Get a site group membership
+      description: |
+        Gets site membership information for group **groupId** on site **siteId**.
+
+      operationId: getSiteGroupMembership
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/siteIdParam'
+        - $ref: '#/parameters/groupIdParam'
+        - $ref: '#/parameters/fieldsParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/SiteMemberEntry'
```

Comment:
I don't think this is going to work "SiteMemberEntry" includes a property called "person" which won't work here as you're returning groups.

We may have to define a new "SiteGroupMemberEntry" object that shows the group and it's role.